data/test/scenarios/wml_tests/ScenarioWML/EventWML/events-test_filters.cfg

```diff
@@ -408,6 +395,23 @@
         {ASSERT ({VARIABLE_CONDITIONAL side_number equals 1})}
         {VARIABLE_OP triggers_on_defense add 1}
     [/event]
+#enddef
+
+#####
+# API(s) being tested: [event][filter_attack],[event][filter_second_attack]
+##
+# Actions:
+# Define events filtering based on weapon specials.
+# Give Bob a poison special iff the opponent has a specific drain special.
+# Give Alice a drain special iff she has a blade attack.
```

Comment:
It's used in a filter_opponent, not a filter_self.

Reply:
Fixed